test/sms_nurse.test.js

```diff
@@ -0,0 +1,37 @@
+var vumigo = require('vumigo_v02');
+var AppTester = vumigo.AppTester;
+var assert = require('assert');
+
+describe("app", function() {
+    describe("for nurse_sms use", function() {
+        var app;
+        var tester;
+
+        beforeEach(function() {
+            app = new go.app.GoNDOH();
+
+            tester = new AppTester(app);
+
+            tester
+                .setup.config.app({
+                    name: 'nurse_sms',
+                    env: 'test',
+                    testing: 'true',
+                    testing_today: 'June 30, 2017 09:07:07'
```

Comment:
You could potentially remove these config values above at the moment if you want a very clean app.